src/ralph/apps.py

```diff
@@ -3,6 +3,8 @@
 
 from django.apps import AppConfig
 
+from ralph.virtual.subscriptions import generate_listeners
```

Comment:
wouldn't it be better to move this import into `ready` method and keep imports at the top of the file in `ralph.virtual.subscriptions`?